### DISCUSSION TITLE
Contribution Graph

### DISCUSSION THREAD

**Discussion by SkMAIL13** [community/community#3959] [Mobile] [answered]
Hello GH, wanna ask, why app haven't contribution graph. I would be good idea too add contribution graph 👍

**Comment by elisealix22** [comment, marked as answer]
Thank you so much for the request! We are tracking this internally and I just added your note to our internal issue. Please keep the feature request and/or bug reports coming! We really appreciate it.

**Reply by SkMAIL13** [reply]
Thanks a lot

**Reply by tusharr1411** [reply]
It's been more than 2 years...

**Reply by minding-myown-business** [reply]
@tusharr1411 yeah.. I came here because I wanted this exact feature, but just a dead end I guess...
